HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/HealthNetworkingService.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  HealthNetworkingService.swift
+//  GiniHealthSDKExample
+//
+//  Created by Nadya Karaban on 23.05.23.
+//
+
+import Foundation
```

Comment:
[suggestion]: we can remove this import since we are not using something in particular from it.

Reply:
we use `NSDate :)`  in the code